internal/build/pom.xml

```diff
@@ -450,7 +450,7 @@
             <dependency>
                 <groupId>org.antlr</groupId>
                 <artifactId>ST4</artifactId>
-                <version>4.3.1</version>
+                <version>4.3.3</version>
```

Comment:
Well, the problem with the lowest version declaration is that in real use cases that means every user is exposed to bugs that have long since been fixed. The vast majority of users will not manage the dependency and would prefer us to have a reasonably up to date version.

I think it is better to treat downgrading as the exception and having the latest as the default - if someone really needs to keep an old version, they are always free to manage it down themselves.

Regarding 4.3.4, because their releases page is out of date :(
https://github.com/antlr/stringtemplate4/releases